### PR TITLE
[15.0][FIX] stock_barcodes_picking_batch: Validation button not displayed

### PR DESCRIPTION
**Pull request by sergio-teruel**
Remove unused code

cc @Tecnativa TT45432

ping @carlosdauden @CarlosRoca13

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
Code review :+1:

File: stock_barcodes_picking_batch/wizard/stock_barcodes_read_picking_batch.py [hunk 1]
```diff
@@ -104,13 +104,6 @@ def get_sorted_move_lines(self, move_lines):
         )
         return move_lines
 
-    def _get_stock_move_lines_todo(self):
```
Comment (rousseldenis) [inline stock_barcodes_picking_batch/wizard/stock_barcodes_read_picking_batch.py:107]: @sergio-teruel Why removing this ?

Comment (sergio-teruel) [reply stock_barcodes_picking_batch/wizard/stock_barcodes_read_picking_batch.py:107]: Is unused code...

**Comment by francesco-ooops** [comment]
@sergio-teruel should this be merged in your opinion?

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

**Pull request merged**